XIVSlothCombo/Combos/PvE/MNK.cs

```diff
@@ -295,29 +295,29 @@ protected override uint Invoke(uint actionID, uint lastComboActionID, float comb
                         {
                             if (IsEnabled(CustomComboPreset.MNK_STUseBrotherhood) 
                                 && Brotherhood.LevelChecked()
-                                && !IsOnCooldown(Brotherhood))
+                                && ActionReady(Brotherhood))
                             {
                                 return Brotherhood;
                             }
 
                             if (IsEnabled(CustomComboPreset.MNK_STUseROF) 
                                 && RiddleOfFire.LevelChecked()
-                                && !IsOnCooldown(RiddleOfFire))
+                                && ActionReady(RiddleOfFire))
                             {
                                 return RiddleOfFire;
                             }
 
                             if (IsEnabled(CustomComboPreset.MNK_STUseROW)
                                 && RiddleOfWind.LevelChecked()
-                                && !IsOnCooldown(RiddleOfWind))
+                                && ActionReady(RiddleOfWind))
                             {
                                 return RiddleOfWind;
                             }
                         }
 
-                        if (PlayerHealthPercentageHp() <= PluginConfiguration.GetCustomIntValue(Config.MNK_ST_SecondWind_Threshold) && IsEnabled(CustomComboPreset.MNK_ST_ComboHeals) && LevelChecked(All.SecondWind) && IsOffCooldown(All.SecondWind))
+                        if (PlayerHealthPercentageHp() <= PluginConfiguration.GetCustomIntValue(Config.MNK_ST_SecondWind_Threshold) && IsEnabled(CustomComboPreset.MNK_ST_ComboHeals) && LevelChecked(All.SecondWind) && ActionReady(All.SecondWind))
                             return All.SecondWind;
-                        if (PlayerHealthPercentageHp() <= PluginConfiguration.GetCustomIntValue(Config.MNK_ST_Bloodbath_Threshold) && IsEnabled(CustomComboPreset.MNK_ST_ComboHeals) && LevelChecked(All.Bloodbath) && IsOffCooldown(All.Bloodbath))
+                        if (PlayerHealthPercentageHp() <= PluginConfiguration.GetCustomIntValue(Config.MNK_ST_Bloodbath_Threshold) && IsEnabled(CustomComboPreset.MNK_ST_ComboHeals) && LevelChecked(All.Bloodbath) && ActionReady(All.Bloodbath))
```

Comment:
`PluginConfiguration.GetCustomIntValue(Config.MNK_ST_SecondWind_Threshold)`
can be replaced with just
`Config.MNK_ST_SecondWind_Threshold`
if it is declared a UserInt class under Config
Also, ActionReady + LevelChecked again

Reply:
> PluginConfiguration.GetCustomIntValue(Config.MNK_ST_SecondWind_Threshold)
can be replaced with just
Config.MNK_ST_SecondWind_Threshold
if it is declared a UserInt class under Config

Not too sure about this tbh because I didn't write this module, so I'll leave this be for now until I get more understanding on this. 

> Also, ActionReady + LevelChecked again

Done 👍🏽